src/frog/io.jl

```diff
@@ -0,0 +1,211 @@
+
+"""
+    module ReportIO
+
+Functions for reading and writing FROG reports.
+"""
+module ReportIO
+
+using ..FROG: FROGReactionReport, FROGObjectiveReport, FROGMetadata, FROGReportData
+
+using ...FBCModelTests: gets
```

Comment:
why 3 dots? I though you need them in pairs to go up a level?

Reply:
well, actually not. :D